app.rb

```diff
@@ -2,4 +2,19 @@
 
 class App < Sinatra::Base
 
-end
\ No newline at end of file
+  get '/' do
+    erb :index
+  end
+
+  get '/new' do
+    erb :create_puppy
+  end
+
+  post '/puppy' do
+    @name = params[:name]
+    @breed = params[:breed]
+    @age = params[:age]
+    erb :display_puppy
```

Comment:
It would be better to use the `Puppy` class here.

Reply:
so like erb :puppy 

or were you referring to 
@puppy.age = params[:puppy][:age]

Reply:
More like 

```ruby
@puppy = Puppy.new(params[:name], params[:breed], params[:age])
```